middleware.js

```diff
@@ -56,7 +56,7 @@ function middleware (api, { baseIriFromRequest, loader, store } = {}) {
     }
   })
 
-  router.use(rdfHandler({ baseIriFromRequest }))
+  router.use(rdfHandler({ baseIriFromRequest, sendTriples: true }))
```

Comment:
Should this be configurable by the middleware options?

Reply:
If we need it, we can still add it later. Let's stay with good defaults for the beginning.